src/routes/(console)/+layout.ts

```diff
@@ -1,71 +1,66 @@
-import { Dependencies } from '$lib/constants';
-import type { Plan } from '$lib/sdk/billing';
-import type { Tier } from '$lib/stores/billing';
 import { sdk } from '$lib/stores/sdk';
 import { isCloud } from '$lib/system';
 import type { LayoutLoad } from './$types';
-import { Query, type Models } from '@appwrite.io/console';
+import { Dependencies } from '$lib/constants';
+import type { Tier } from '$lib/stores/billing';
+import type { Plan, PlanList } from '$lib/sdk/billing';
+
+export const load: LayoutLoad = async ({ depends, parent }) => {
+    let { organizations } = await parent();
 
-export const load: LayoutLoad = async ({ params, fetch, depends, parent }) => {
-    await parent();
     depends(Dependencies.RUNTIMES);
     depends(Dependencies.CONSOLE_VARIABLES);
     depends(Dependencies.ORGANIZATION);
 
-    const prefs = await sdk.forConsole.account.getPrefs();
-
     const { endpoint, project } = sdk.forConsole.client.config;
-    const versionPromise = fetch(`${endpoint}/health/version`, {
-        headers: {
-            'X-Appwrite-Project': project
-        }
-    }).then((response) => response.json() as { version?: string });
+    const [preferences, plansArray, orgs, versionData, consoleVariables] = await Promise.all([
+        sdk.forConsole.account.getPrefs(),
+        isCloud ? sdk.forConsole.billing.getPlansInfo() : null,
+        // reuse from parent if available, else load fresh.
+        // know that the parent loads these orgs conditionally!
+        !organizations
+            ? isCloud
+                ? sdk.forConsole.billing.listOrganization()
+                : sdk.forConsole.teams.list()
+            : organizations,
```

Comment:
do we even need this if parent has it already? the dependency is invalidated anyway when its needed 👍🏻

Reply:
done.